rllib/core/models/configs.py

```diff
@@ -92,16 +92,36 @@ class MLPHeadConfig(ModelConfig):
         ReLU()
         Linear(8, 2)
 
+    Example:
+
+        Configuration:
+        input_dims = [2]
+        hidden_layer_dims = [10]
+        hidden_layer_activation = "silu"
+        hidden_layer_use_layernorm = True
+        output_dims = [2]
+
+        Resulting stack in pseudocode:
+        Linear(2, 10)
+        LayerNorm()
+        SiLU()
+        Linear(10, 2)
+
     Attributes:
         hidden_layer_dims: The sizes of the hidden layers.
         hidden_layer_activation: The activation function to use after each layer (
             except for the output).
+        hidden_layer_use_layernorm: Whether to insert a LayerNorm functionality
```

Comment:
Added to all primitives (MLP and CNN):
* option to switch on layernorm'ing in between layers
* use bias or not